tests/kernel/interrupt/src/interrupt_util.h

```diff
@@ -60,6 +60,24 @@ static inline void trigger_irq(int irq)
 #endif
 }
 
+#elif defined(CONFIG_GIC)
```

Comment:
Is this supposed to work for ARM-only? If so, some comment or even and ifdef CORTEX_A, R would help

Reply:
well, `CONFIG_GIC` is defined to be (ARM) Cortex-only:
https://github.com/zephyrproject-rtos/zephyr/blob/2b441732ff3ba31b1c3bf73497b0aab0279d8011/drivers/interrupt_controller/Kconfig.gic#L6-L8